inference/core/interfaces/http/orjson_utils.py

```diff
@@ -96,6 +110,9 @@ def serialise_dict(elements: Dict[str, Any]) -> Dict[str, Any]:
     return serialised_result
 
 
+@deprecated(
+    reason="Function contains_image(...) will be removed from `inference` end of Q3 2024"
```

Comment:
Is this just because we don't use it anymore?

Reply:
yes

Reply:
I just dont want to put it into thrash without notice to people using it